bin/sofa_utils.py

```diff
@@ -0,0 +1,27 @@
+import json
+import pandas as pd 
+import itertools
+import os
+from collections import defaultdict, OrderedDict
+
+def parse_pyflame(file_path):
+
+    func_dict = {}
+    
+    with open(file_path, 'r') as f:
+        for time_stamp, funct_stack in itertools.zip_longest(*[f] * 2):
+            funct_stack = funct_stack.replace('\n', '').replace(';', '<br>')
+            func_dict[int(time_stamp)] = funct_stack
+        func_dict = OrderedDict(func_dict)
```

Comment:
`dict` is not ordered in <Python3.6. It's insertion-ordered in **C**Python3.6. It's insertion-ordered in Python3.7+.

Sofa supports Python 3.6, not only(?) CPython3.6.
Thus, converting an unordered dict to `OrderedDict` doesn't make sense.